KenticoCloud.ContentManagement.Helpers.Tests/EditLinkBuilderTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+
+using KenticoCloud.ContentManagement.Helpers.Configuration;
+
+using Xunit;
+
+namespace KenticoCloud.ContentManagement.Helpers.Tests
+{
+    public class EditLinkBuilderTests
+    {
+        private readonly string _itemId = "1cdaa8ef-cb2b-4f82-82e9-45467b2e01b9";
+        private readonly string _language = "some-Language-Codename";
+
+        [Fact]
+        public void BuildEditItemUrl_ValidInput_ReturnsValidUrl()
+        {
+            var options = new ContentManagementHelpersOptions
+            {
+                ProjectId = "14dc0cf8-6cc1-4f20-8e2e-0b5edea89e43"
+            };
+
+            var underTest = new EditLinkBuilder(options);
+            var result = underTest.BuildEditItemUrl(_language, _itemId);
+
+            var expected = "https://app.kenticocloud.com/goto/edit-item/item/1cdaa8ef-cb2b-4f82-82e9-45467b2e01b9/variant-codename/some-Language-Codename/project/14dc0cf8-6cc1-4f20-8e2e-0b5edea89e43";
```

Comment:
Is there any specific reason to have "item" twice in the URL?
It is also weird to have project ID at the end, it should be ordered by the granularity of it, shouldn't it?

Reply:
1) to have word "item" in path is necessary for future compatibility
2) order of project - variant - item is fixed now